test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+let expect = require('expect'),
+    jsonurl = require('.'),
```

Comment:
I had some difficulties running it with `require('.')`. Changing it to `require('./index')` seems to help. Or possibly move this file to a `/test` folder?